.cirrus.yml

```diff
@@ -30,8 +30,10 @@ env:
     PRIOR_FEDORA_NAME: "fedora-35"
     UBUNTU_NAME: "ubuntu-2110"
 
-    # Google-cloud VM Images
+    # Image identifiers
     IMAGE_SUFFIX: "c6211193021923328"
+    FEDORA_AMI_ID: "ami-04f37091c3ec43890"
```

Comment:
@ashley-cui - FYI ^^^^^ is the image to use in EC2 in case you didn't write it down :wink: